components/decorateMapComponent.js

```diff
@@ -32,7 +32,9 @@ export const createNotSupportedComponent = message => () => {
   return null;
 };
 
-export const googleMapIsInstalled = !!NativeModules.UIManager[getAirMapName(PROVIDER_GOOGLE)];
+export const googleMapIsInstalled = Boolean(
+  NativeModules.UIManager[getAirMapName(PROVIDER_GOOGLE)]
+);
```

Comment:
Let's make it `export const googleMapIsInstalled = !!!!NativeModules.UIManager[getAirMapName(PROVIDER_GOOGLE)];` while we're at it 😉